designsafe/apps/workspace/static/designsafe/apps/workspace/scripts/controllers/application-form.js

```diff
@@ -220,7 +227,8 @@
             jobData.processorsPerNode = jobData.nodeCount * ($scope.data.app.defaultProcessorsPerNode / $scope.data.app.defaultNodeCount);
           }
 
-          $scope.data.submitting = true;
+          if ($scope.data.up) {

```

Comment:
Rather than have this here, make the form readonly if the system is down (lines 108, 125 & 132). Maybe make a readonly attribute for clarity.